src/app/calculator/furnaces/flue-gas/energy-form/energy-form.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { EnergyData } from '../../../../shared/models/phast/losses/chargeMaterial';
+
```

Comment:
Provide the form in FlueGasModule, it's working now because of the providedIn but for consistency purposes add it to the module like we are doing everywhere else